range-diff.c

```diff
@@ -106,6 +108,17 @@ static int read_patches(const char *range, struct string_list *list,
 			continue;
 		}
 
+		if (util == NULL) {
```

Comment:
In Git's source code, we usually write `if (!util)` in such instances. Maybe you want to do the same here?

Reply:
Done in b3384880c7. Btw, `git grep '== NULL'` returns 298 matches.